metrics/mixin/alerts/blocklist.libsonnet

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
After adding this file, we also need to generate YAML that actually gets applied to clusters.
`make gen-prometheus-rule` should help.